shell/probe/install.ps1

```diff
@@ -11,7 +11,15 @@ param(
 )
 
 [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
-$PRELUDE_API=if ($Env:PRELUDE_API) { $Env:PRELUDE_API } else { "https://api.preludesecurity.com" }
+
+function FromEnv { param ([string]$envVar, [string]$default)
+    $envVal = [Environment]::GetEnvironmentVariable($envVar, "Process")
+    if($envVal) {return $envVal}
```

Comment:
if it's only a process-scoped variable, should you also set the user-scoped variable of the same name so the installed probe will work?

Reply:
or could update the cmd we're passing the scheduled task to set the process-scoped env variable before starting raindrop

Reply:
The install script sets the PRELUDE_TOKEN at User-level assuming that the script is not modified by a user, I don't see how the PRELUDE_TOKEN at Process-level would override User-level

Reply:
was thinking about the PRELUDE_API variable - just so that the probe you install is talking to the right environment

Reply:
Ah good catch!